diagnostic.md

```diff
@@ -32,7 +32,7 @@ _Replace this text with your answer._
 
 Is the value of this expression `true` or `false`?  Explain your answer.
 
-_Replace this text with your answer._
+In Booleon logic this would resolve to false, I know this because I ran the above in JSbin. I cannot currently explain the reasoning.
```

Comment:
It does evaluate to false.  The left side (false || true) evaluates to true because the binary operator || yields true if at least one operand is true.  The right side (false && true) evaluates to false because && yields true only if both operands are true.  This results in the expression being reduced to true && false, which evaluates to false.